advanced/Scripts/updatecheck.sh

```diff
@@ -32,21 +32,63 @@ function get_local_branch() {
 }
 
 function get_local_version() {
-# Return active branch
-cd "${1}" 2> /dev/null || return 1
-git describe --long --dirty --tags || return 1
+  # Return active branch
+  cd "${1}" 2> /dev/null || return 1
+  git describe --long --dirty --tags || return 1
+}
+
+function extract_dnsmasq_version() {
+  # Return version of system-wide dnsmasq daemon
+  dnsmasq -v | awk '/Dnsmasq version/{print $3}'
```

Comment:
This might not work for other locales: https://discourse.pi-hole.net/t/more-web-site-slow/7570/5?u=mcat12